Rakefile

```diff
@@ -26,7 +26,7 @@ begin
   require_relative "tasks/announce"
   require_relative "tasks/docs"
   require_relative "tasks/spellcheck"
-  require_relative "lib/chef/dist"
+  require_relative "chef-utils/lib/chef-utils/dist"
```

Comment:
you might want to slack the defined? bit in here to avoid that chefstyle issue when we bump that in chef/chef